docs/project-plan.md

```diff
@@ -3,162 +3,332 @@
 
 ## Index
 1. Product Statement
-2. Functional Requirements
-3. Value Stories
-4. Releases
-5. Roles
+2. Value Stories
+3. Releases
+4. Roles
+5. Sprints & Meeting Schedule
 
-## 1.0 Product Statement
+## 1.0 - Product Statement
+University is full of hard work and tough challenges. Unfortunately, this experience is made more difficult without a centralised source for course information. 'Smart Course' is our solution to this problem.
 
-Smart Course is a web application that allows university students and lecturers to communicate information about courses they are involved in.
+'Smart Course' is a web application that allows university students to ask questions, evaluate and share general information about courses all year long. Whether it be in the midst of subject selection time, or the week before finals, you can find out what you need to know!
 
-## 2.0 Functional Requirements
+## 2.0 - Value Stories
 
-TBD
-
-## 3.0 Values Stories
-
-### 3.1 Course Questions & Answers [EPIC, THEME]
-Students and teachers should be able to share information about a course.
+### 2.1 Course Questions & Answers
+Students and teachers should be able to share specific information about a course.
 
 #### Ask Questions
-A student should be able to ask questions about a course they are interested in.
-
-#### Answer/Comment on Questions
-A student or lecturer should be able to answer a question previously posted about a course.
-
-#### Upvoting/Rating
-As a user I want to express the usefulness or relevance of a question or answer by upvoting it.
-
-#### Accepted answer
-As a user I want to quickly see the most relevant or upvoted answer for a given question.
-
-#### Question priority
-As a question answerer I want to quickly see the most relevant or popular questions.
-
-#### Answerer Reputation
-As a reliable contributer I want my answers to be given more weight.
-
-#### Answerer verification
-As someone looking at the answer/s to a question, I want to see which answerers are teachers (lecturers, TAs) for the course and which answerers have done the course (or are currently doing it).
+As a student, I want to be able to ask a question about a course, so that I have access to information I didn't have previously.
+ * Estimate: 6
```

Comment:
Not sure if we can estimate these very well in here, but I spose it's just for show. Agree need to formalise the Stories that make it into the MVP. (There is a release tag).